service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/CloneGcpWorkspaceFlight.java

```diff
@@ -29,5 +29,8 @@ public CloneGcpWorkspaceFlight(FlightMap inputParameters, Object applicationCont
 
     addStep(new LaunchCloneAllResourcesFlightStep(), RetryRules.cloud());
     addStep(new AwaitCloneAllResourcesFlightStep(), RetryRules.cloudLongRunning());
+
+    addStep(new FindEnabledApplicationStep(flightBeanBag.getApplicationDao()), RetryRules.cloud());
```

Comment:
Instead of adding these steps, I think it's cleaner to change `FindResourcesToCloneStep` to return enabled application resources.